src/main/java/base/Configuration.java

```diff
@@ -0,0 +1,16 @@
+package base;
+
+public class Configuration {
```

Comment:
maybe another name like `warnings`  or sth like that could be somehow better.

Reply:
Possibly you are right but I am thinking this class as having other entries as well and not just warnings. I would say let's leave it like this for now and come back to it in the future. by the way there is a dedicated library, ```log4j``` I think is called for logging in Java...If you insist I can do as say it is not a big deal...

Reply:
let's keep it this way for now.